examples/web-react/index.html

```diff
@@ -1,60 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8" />
-  <title>Spirit Web React Demo</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-
-  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet" />
-
-  <!-- Load Spirit CSS. -->
-  <link rel="stylesheet" href="built/components.min.css" />
-
-  <!-- Load demo app. -->
-  <script src="built/demo.js" defer></script>
-
-  <style>
-    body {
-      margin: 0;
-      padding: 0;
-      font-family: Inter, sans-serif;
-    }
-
-    h1 {
-      margin-block: 3rem;
-      font-size: 4rem;
-    }
-
-    h2 {
-      margin-top: 3rem;
-      margin-bottom: 1.5rem;
-      font-size: 2rem;
-    }
-
-    header,
-    main {
-      padding: 1.5rem 8vw;
-    }
-
-    @media (min-width: 960px) {
+  <head>
+    <meta charset="UTF-8" />
+    <title>Spirit Web React Demo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet" />
+
+    <!-- Load Spirit CSS. -->
+    <link rel="stylesheet" href="built/components.min.css" />
+
+    <!-- Load demo app. -->
+    <script src="built/demo.js" defer></script>
+  </head>
+  <body>
+    <h1>Spirit Web React Demo</h1>
+
+    <style>
```

Comment:
Why has `style` moved from `head` to `body`? I don't wish any format checker to do such changes…

Moreover, it's even invalid:

> Contexts in which this element can be used:
>  - Where metadata content is expected.
>  - In a noscript element that is a child of a head element.

https://html.spec.whatwg.org/multipage/semantics.html#the-style-element

https://stackoverflow.com/a/50154225

Reply:
Sorry about this. It was a very badly done rebase and I did not notice. Now it is all right.